pkg/runner/cluster_swarm.go

```diff
@@ -142,6 +143,13 @@ func (*ClusterSwarmRunner) Run(input *api.RunInput, ow io.Writer) (*api.RunOutpu
 		Internal:   true,
 		Attachable: true,
 		Scope:      "swarm",
+		IPAM: &network.IPAM{
+			Driver: "default",
+			Config: []network.IPAMConfig{{
+				Subnet:  "192.168.0.0/16",
+				Gateway: "192.168.0.1",
```

Comment:
FYI, I'm setting permanent network ranges with public and private IP addresses in https://github.com/ipfs/testground/pull/271.

(doesn't block this PR, I'll rebase that one).